jflex/pom.xml

```diff
@@ -191,6 +191,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-jar-to-lib</id>
+            <phase>install</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/lib</outputDirectory>
```

Comment:
Why do we need to copy the uberjar in the lib directory? For Maven users, this seems really unnecessary.

I do remember doing something similar for running ant examples in
https://github.com/jflex-de/jflex/blob/8657dfb65629dd29252884e92ea5407404b7c8cf/scripts/test-examples.sh#L32

The difficulty, I guess, is to make ant work both from source code (where the uberjar is in target) and from the release zip (where the uberjar is in lib)

Reply:
It’s mostly for being able to run the bin/jflex scripts during dev, eg for testing Makefiles and others the same way one would in the distribution after release. For maven it is indeed unnecessary, but we want people to be able to use other build systems after release. Having the jar in lib up-to-date in dev just makes that a bit easier.

Reply:
(my long-winded way of saying yes, you're right :-))